.cirrus.yml

```diff
@@ -18,6 +18,16 @@ container:
 
 
 env:
+  # Skip specific tasks by name.
```

Comment:
Would be useful to say something like this:

"Skip specific tasks by name.  Set to a non-empty string to skip."